tests/components/advantage_air/test_sensor.py

```diff
@@ -0,0 +1,126 @@
+"""Test the Advantage Air Sensor Platform."""
+
+from json import loads
+
+from homeassistant.components.advantage_air.const import DOMAIN
```

Comment:
Just to keep it tidy :)
```suggestion
from homeassistant.components.advantage_air.const import DOMAIN as ADVANTAGE_AIR_DOMAIN
```